package.json

```diff
@@ -43,13 +43,15 @@
     "install": "node install.js",
     "npm": "npm link npm",
     "test": "eslint . && ./tests/unittests.sh && ./tests/integration.sh",
```

Comment:
```suggestion
     "test": "npm run lint && ./tests/unittests.sh && ./tests/integration.sh",
```

Reply:
@lundibundi, this has nothing to do with this PR.